spec/factories/casa_orgs.rb

```diff
@@ -13,14 +13,9 @@
       logo { Rack::Test::UploadedFile.new(Rails.root.join("spec", "fixtures", "org_logo.jpeg")) }
     end
 
-    trait :with_placement_types do
-      transient { placement_names { ["Reunification", "Adoption", "Foster Care", "Kinship"] } }
-
-      after(:create) do |org, evaluator|
-        evaluator.placement_names.each do |name|
-          org.placement_types.create!(name: name)
-        end
-      end
```

Comment:
arg... merge conflicts that I ignored (-X theirs). dang. will rebase again with more care.